tests/unit/test_property.py

```diff
@@ -1,5 +1,35 @@
-from conftest import newPropertyName, newPropertyAddress
+from models.property import PropertyModel
 
-def test_new_property(new_property):
-    assert new_property.name == newPropertyName
-    assert new_property.address == newPropertyAddress
\ No newline at end of file
+name = "new_property"
+address = "789 Ten St."
+city = "Portland"
+unit = "101"
+state = "OR"
+zipcode = "97207"
+propertyManagerIDs = [5, 6]
+dateAdded = "2020-04-12"
+archived = 0
+
+
+def test_new_property(test_database):
```

Comment:
FYI: This test doesn't really have much value. It's testing a plain python init function. We wouldn't even need to have the database if it weren't for the set_property_managers function. 

We could set the parameters to:
```
name = 0
address = 0
city = 0
unit = 0
state = 0
zipcode = 0
propertyManagerIDs = []
dateAdded = 0
archived = 0
```

And everything still passes. Hence, the implementation for the init function is being tested. We do need some unit tests for the property model. We can leave this in for now, and I will update the refactoring issues to add the appropriate unit tests on the model.

Reply:
I can remove this for now.